test/integration/sequelize.test.js

```diff
@@ -273,6 +273,23 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
       return this.sequelize.query(this.insertQuery);
     });
 
+    it('executes a query if a placeholder value is an array', function() {
```

Comment:
Maybe adding a unit test to check for the output of the query generator as well might be good?

Reply:
@eseliger, done.